beaglebone.coffee

```diff
@@ -12,9 +12,9 @@ postProvisioningInstructions = [
 
```

Comment:
Change this commit and remove these changes from beaglebone.coffee file

Reply:
This is done, @floion